app/api/v2/commercial/entities/account.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module API::V2::Commercial
+  module Entities
+    class AccountBase < API::V2::Entities::Base
+      expose :id,
+             documentation: {
+               type: 'Integer',
+               desc: 'Organization ID'
+             }
+
+      expose :oid,
+             documentation: {
+               type: 'String',
+               desc: 'Organization OID'
+             }
+
+      expose :name,
+             documentation: {
+               type: 'String',
+               desc: 'Organization Account Name'
+             }
+    end
+
+    class Account < AccountBase
+      expose :uids do |member|
+        member.memberships.map { |m| m.user.uid }
+      end
+    end
+
+    class OrganizationAccount < AccountBase
+      expose :users do |member|
```

Comment:
why we call field  users, and return length?

Reply:
This model for organization management for Tower to show users in each subunit

![image](https://user-images.githubusercontent.com/241133/120187404-efb04680-c23e-11eb-944f-108573acf71b.png)